libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/instruction/InstructionView.java

```diff
@@ -108,6 +108,7 @@ public InstructionView(Context context, @Nullable AttributeSet attrs) {
 
   public InstructionView(Context context, @Nullable AttributeSet attrs, int defStyleAttr) {
     super(context, attrs, defStyleAttr);
+    locale = context.getResources().getConfiguration().locale;
```

Comment:
Replace with extraction from `SharedPreferences`

Reply:
@danesfeder I was thinking just pass the context to `DistanceUtils`, and then `DistanceUtils` can get the locale from `SharedPreferences`

Reply:
@devotaaabel yeah that sounds good to me - just need to make sure we aren't calling `DistanceUtils` somewhere where we don't have `Context`.  Or have a backup method that can be called without context

Reply:
@danesfeder To localize the strings like Eric said we'll need the context, but it shouldn't be an issue anyway